internal/backend/backend_test.go

```diff
@@ -43,16 +43,48 @@ func testForStorageBackends(t *testing.T, test func(t *testing.T, storage Storag
 	}
 }
 
-func noError(t *testing.T, err error) {
+func noError(t *testing.T, err error, description string) {
```

Comment:
without description, it was difficult to track where errors came from

Reply:
Same as below. Should we mark this as a helper?

Reply:
Yes! In fact, i didnt know about the helper functionality until two weeks ago. I will incorporate, minimizing or removing the usage of descriptions.